liquidcore/site/settings.py

```diff
@@ -11,8 +12,8 @@ def bool_env(value):
 DEBUG = bool_env(os.environ.get('DEBUG'))
 SECRET_KEY = os.environ.get('SECRET_KEY')
 liquid_http_protocol = os.environ.get('LIQUID_HTTP_PROTOCOL', 'http')
-LIQUID_DOMAIN = os.environ['LIQUID_DOMAIN']
-LIQUID_TITLE = os.environ['LIQUID_TITLE']
+LIQUID_DOMAIN = os.environ.get("LIQUID_DOMAIN", "null")
```

Comment:
I'd like both of these to default to `None` if they are not set; having random `null` strings pop up in the UI is a pain to debug later. This means running `os.getenv('LIQUID_DOMAIN')`

I told you about the `null` default related to the LIQUID_APPS variable, so the `"null"` string gets evaluated as JSON and still returns `None`.